# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/tests/non262/extensions/typedarray.js
# Commit: f207beed6252
# Full Hash: f207beed6252d85e5a64cc768eaef9d8a47b4912
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2022-10-28 04:43:55
# Regressor Bug: 1797755
# File Overlap Count: 1
# Description:
#   Bug 1797755 - Part 4: Remove option to set maximum mark stack capacity in release builds r=sfink
#   
#   Currently we have a GC parameter that allows setting a maximum mark stack
#   capacity. This is only ever used by test code, not in the browser. This
#   requires extra unnecessary work in release builds if we move to a different
# ==============================================================================

diff -r 4746d15d7a9f -r f207beed6252 js/src/tests/non262/extensions/typedarray.js
--- a/js/src/tests/non262/extensions/typedarray.js	Thu Oct 27 18:23:03 2022 +0000
+++ b/js/src/tests/non262/extensions/typedarray.js	Thu Oct 27 18:23:03 2022 +0000
@@ -201,7 +201,9 @@
             }
         }
 
-        gcparam('markStackLimit', 200);
+        if (typeof setMarkStackLimit === 'function') {
+            setMarkStackLimit(200);
+        }
         var forceOverflow = [ buffer ];
         for (let i = 0; i < 1000; i++) {
             forceOverflow = [ forceOverflow ];
